internal/jobs/ensure_pubkey_on_aws_job_test.go

```diff
@@ -112,6 +112,8 @@ func TestHandleEnsurePubkeyOnAWS(t *testing.T) {
 		pkrList, err := pkDao.UnscopedListResourcesByPubkeyId(ctx, pk.ID)
 		require.NoError(t, err)
 		assert.Equal(t, 1, len(pkrList))
+		assert.Equal(t, "awsPreviouslyImportedTag", pkrList[0].Tag)
+		assert.Equal(t, "key-1", pkrList[0].Handle)
```

Comment:
I don't see "key-1" in the code can you please explain? :)

Reply:
the `key-1` is automatically calculated as a serie in the stub, but might be better not to rely on it :thinking: